test/shape_test.cpp

```diff
@@ -64,6 +64,15 @@ TEST_CASE(test_shape_standard)
     EXPECT(not s.broadcasted());
 }
 
+TEST_CASE(test_shape_standard_stride)
+{
+    migraphx::shape s{migraphx::shape::float_type, {1, 2048, 1, 1}, {2048, 1, 2048, 2048}};
+    EXPECT(not s.standard());
+    EXPECT(s.packed());
+    EXPECT(s.transposed());
+    EXPECT(not s.broadcasted());
+}
```

Comment:
This test is bot related to this PR. It should go in a different PR.

Reply:
ok, reverted